docs/docs/lib/flutter.mdx

```diff
@@ -28,18 +19,16 @@ To create a GrowthBook SDK instance, use `GBSDKBuilderApp`. Then, you can evalua
 
 ```dart
 // User attributes for targeting and assigning users to experiment variations
-val attrs = HashMap<String, Any>()
-attrs.put("id", "123")
-attrs.put("env", "dev")
-attrs.put("betaUser", true)
+final attrs = <String, dynamic>{};
```

Comment:
This variable is plural (`attrs`), but it's referenced as singular everywhere else (`attr`)